Child theme assets (#1731)
PrestaShop/docs: themes/getting-started/theme-yml.md

        description: Great for CMS pages to show advertisements on the side
```

## Parent / child settings

{{% notice note %}}

> Find mode informations about [Theming inheritance - Parent/child themes]({{<relref "/8/themes/reference/template-inheritance/parent-child-feature">}})

Reviewer: ```suggestion
Find more information about [Theming inheritance - Parent/child themes]({{<relref "/8/themes/reference/template-inheritance/parent-child-feature">}})
```

{{% /notice %}}

To set the parent theme, set the parent key to the theme's name: 

```yaml
parent: classic
```

### use_parent_assets 

> From {{< minver v="8.0" >}} , the setting `use_parent_assets` is functional.

Reviewer 2: As we are inside the 8 part of the doc, it's needed to say that the minver is 8 ?

Author: Since the option was available in 1.7 but not working, i thought it was needed ?

> From {{< minver v="8.0" >}} , the setting `use_parent_assets` is functional.

Reviewer: ```suggestion
Starting from {{< minver v="8.0" >}} , the setting `use_parent_assets` is now fully working and correctly loads assets from the parent theme.
```

```yaml
assets:
  use_parent_assets: true
```

> When set to `true`, some additional variables are changed through `Smarty`. 

Reviewer: ```suggestion
When set to `true`, some additional variables are changed for `Smarty` templates. 
```

- `theme_assets` is replaced with the parent theme assets URI, 

> - `img_url`, `css_url`, `js_url` are replaced also with the parent theme corresponding URIs,

Reviewer: ```suggestion
- `img_url`, `css_url`, `js_url` are replaced with the parent theme corresponding URIs,
```

> And some other variables are created: 

Reviewer: ```suggestion
some other variables are available: 
```
